sap-concur/4.0/modules/ROOT/pages/index.adoc

```diff
@@ -76,32 +76,15 @@ image::sap-concur-config-design.png[config connector]
 |Authorization| A Concur access token.
 |===
 +
-. Create an access token for your Concur instance by specifying `+https://www.concursolutions.com:443/net2/oauth2/accesstoken.ashx+` in Postman with the GET method.
-. Configure Basic Auth with your username and password.
+. Create an access token for your Concur instance. Creating an access token for your Concur instance by specifying `+https://www.concursolutions.com:443/net2/oauth2/accesstoken.ashx+` in Postman with the GET method is https://developer.concur.com/api-reference/authentication/authorization-pre-2017.html[deprecated]. To create an access token, use one of the following authentication methods:
```

Comment:
```suggestion
. Create an access token for your Concur instance. Creating an access token for your Concur instance by specifying `+https://www.concursolutions.com:443/net2/oauth2/accesstoken.ashx+` in Postman with the GET method is https://developer.concur.com/api-reference/authentication/authorization-pre-2017.html[deprecated]. To create an access token, use one of the following authentication methods:
```

Reply:
"with the GET method is" does not make sense. 
Do you mean "using the GET method ..." or "when the GET method is..."?